native/src/components/MapView.tsx

```diff
@@ -87,6 +87,14 @@ const MapView = ({
     setFollowUserLocation(true)
   }, [onRequestLocationPermission, setFollowUserLocation])
 
+  const onUserTrackingModeChange = (
+    event: MapLibreGLEvent<'usertrackingmodechange', { followUserLocation: boolean }>
+  ) => {
+    if (!event.nativeEvent.payload.followUserLocation) {
```

Comment:
To avoid artifacts in iOS. See video and explanation [in this comment](https://github.com/digitalfabrik/integreat-app/pull/970#discussion_r1265139524). I can also remove this and look how it could be fixed in the library.